pkg/kubelet/prober/manager.go

```diff
@@ -207,6 +207,23 @@ func (m *manager) UpdatePodStatus(podUID types.UID, podStatus *api.PodStatus) {
 		}
 		podStatus.ContainerStatuses[i].Ready = ready
 	}
+	// init containers are ready if they have exited with success or if a readiness probe has
+	// succeeded.
+	for i, c := range podStatus.InitContainerStatuses {
```

Comment:
I definitely wanted to set the expectation they would not be invoked,
but I want to enable them over time.  For alpha, having them disabled
via the API and validations didn't seem unreasonable.